connection.js

```diff
@@ -20,7 +21,7 @@ function Connection(duplex, opts) {
 
   opts = opts || {}
 
-  var inStream  = generateStream()
+  var inStream  = writeToStream()
```

Comment:
No, `generateStream` isn't being used inside mqtt-connection anymore. It's part of the public API so I left it as is. Do you want me to remove it the whole file or just the reference to it from this file?